SPECS/geos/geos.spec

```diff
@@ -0,0 +1,385 @@
+Summary:        GEOS is a C++ port of the Java Topology Suite
+Name:           geos
+Version:        3.11.1
```

Comment:
Is this the latest version? If not at least check whether this version has any CVEs. We shouldn't introduce any package with CVEs existing.

Reply:
This is the latest version.